serverlessworkflow/modules/ROOT/pages/cloud/operator/configuring-workflow-eventing-system.adoc

```diff
@@ -0,0 +1,315 @@
+
+.Platform scoped eventing system configuration example
+
+include::common/platform-scoped-eventing-system-configuration-example.adoc[]
+
+[#workflow-scoped-eventing-system-configuration]
+== Workflow-scoped Eventing system configuration
+
+A workflow-scoped eventing system configuration provides the ability to do a fine-grained configuration of the eventing system for the events `produced` and `consumed` by a workflow.
+
+To configure a workflow-scoped eventing system you must use the fields `spec.sink.ref` and `spec.sources[]` in the `SonataFlow` CR.
+
+[#outgoing-eventing-system-configuration]
+=== Outgoing Eventing system configuration
+
+To configure a workflow-scoped eventing system for the workflow outgoing events, you must use the field `spec.sink.ref` in the `SonataFlow` CR.
+
+This information signals the SonataFlow Operator to automatically link the current workflow, to produce the events by using that Broker.
+That includes, the {product_name} system events, and the workflow business events.
+
+The following `SonataFlow` CR fragment shows an example of such configuration:
+
+.Workflow-scoped outgoing eventing system configuration example
+[source, yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlow
+metadata:
+  name: example-workflow
+  namespace: example-workflow-namespace
+  annotations:
+    sonataflow.org/description: Example Workflow
+    sonataflow.org/version: 0.0.1
+    sonataflow.org/profile: preview
+spec:
+  sink:
+    ref:
+      name: outgoing-example-broker <1>
+      namespace: outgoing-example-broker-namespace <2>
+      apiVersion: eventing.knative.dev/v1
+      kind: Broker
+  flow: <3>
+    start: ExampleStartState
+    events: <4>
+      - name: outEvent1 <5>
+        source: ''
+        kind: produced
+        type: out-event-type1 <6>
+
+    ... <7>
+----
+
+<1> Name of the Knative Eventing Broker to use for all the events produced by the workflow, including the {product_name} system events.
+<2> Optional: Defines the namespace of the Knative Eventing Broker. Defaults to the SonataFlow namespace. In general, we recommend creating the Knative Eventing Broker in the same namespace as the SonataFlow.
+<3> Flow definition field in the `SonataFlow` CR.
+<4> Events definition field in the `SonataFlow` CR.
+<5> Example of an outgoing event `outEvent1` definition.
+<6> Event type for the outgoing event `outEvent1`
+<7> Only a fragment of the workflow is shown for simplicity.
+
+[#incoming-eventing-system-configuration]
+=== Incoming Eventing system configuration
+
+To configure a workflow-scoped eventing system for the workflow incoming events, you must use the field `spec.sources[]` in the `SonataFlow` CR.
+And, you must add an entry in the array, for every event type that requires an individual configuration.
+
+This information signals the SonataFlow Operator to automatically link the current workflow, to potentially `consume` the events from different Brokers, depending on the event type.
```

Comment:
Not necessarily a broker, but any valid Knative Event Source.

Reply:
Really not. Here the only thing we support are Brokers.